index.html

```diff
@@ -0,0 +1,285 @@
+          <br>
+          <button type="button" id='show-me-btn' name="showMe" class='maven ivory-text'>Show Me My New Friends!</button>
+        </form>
+      </div>
+    </div>
+    </section>
+
+
+    <section id='narrowResults' class='section-wrapper'>
+      <h1 id='noMatches' class='maven ivory-text'>
+        Sorry there were no pets matching your criteria. Please choose different options and search again.
+      </h1>
+    </section>
+    <section id="Animal_Detail" class='section-wrapper'>
+      <div id="Animal_Detail_Wrapper"></div>
+      <div>
+        <button id='save-pet-btn'>Save For Later</button>
+        <button id='back-search-btn'>Back to Results</button>
+      </div>
+    </section>
+    <div>
+    <section id="resources" class='section-wrapper maven ivory-text'>
+      <h1>Pet Resources</h1>
+          <h2>Rehoming a Special-Needs Pet</h2>
+            <p><a href="http://bestfriends.org/resources/rehoming-special-needs-pet" class='maven ivory-text underline'>bestfriends.org</a></p>
+            <p><a href ="http://www.petswithdisabilities.org" class='maven ivory-text underline'>petswithdisabilities.org</a></p>
+            <p><a href="http://vth.vetmed.wsu.edu/good-samaritan" class='maven ivory-text underline'>vth.vetmed.wsu/edu</a></p>
+          <h2>Pet Supplies for Special-Needs Pets</h2>
+            <p><a href="http://www.handicappedpets.com/" class='maven ivory-text underline'>handicappedpets.com</a></p>
+            <p><a href="http://hpets.org/" class='maven ivory-text underline'>hpets.org</a></p>
+            <p><a href="http://www.k9carts.com/" class='maven ivory-text underline'>k9carts.com</a></p>
+            <p><a href="http://useddogwheelchairs.com/" class='maven ivory-text underline'>useddogwheelchairs.com</a></p>
+          <h2>Pet Insurance</h2>
+            <p><a href="http://www.consumersadvocate.org/pet-insurance/best-pet-insurance" class='maven ivory-text underline'>consumersadvocate.org</a></p>
+            <p><a href="http://www.healthypawspetinsurance.com/" class='maven ivory-text underline'>Healthy Paws Pet Insurance</a></p>
+            <p><a href="http://www.gopetplan.com/" class='maven ivory-text underline'>Pet Plan</a></p>
+            <p><a href="http://www.embracepetinsurance.com/" class='maven ivory-text underline'>Embrace Pet Insurance</a></p>
+          <h2>Financial Assistance</h2>
+            <p><a href="http://www.handicappedpets.com/pet-services/financial-aid-for-pets" class='maven ivory-text underline'>Handicapped Pets</a></p>
+            <p><a href="http://www.carecredit.com/vetmed" class='maven ivory-text underline'>Care Credit</a></p>
+    </section>
+    </div>
+
+    <section id="favoritesSection" class='section-wrapper'>
+      <h1 class='maven ivory-text'>Favorite Pets</h1>
+      <div id="savedPets"></div>
+    </section>
+
+    <section id="about" class='section-wrapper'>
```

Comment:
The about us page is nice. There should have been some additional thought toward the aspect size of those images. Some are large, and some are small?